mix.exs

```diff
@@ -36,8 +36,10 @@ defmodule ExMachina.Mixfile do
     [
       {:ex_doc, "~> 0.14", only: :dev},
       {:earmark, ">= 0.0.0", only: :dev},
-      {:ecto, "~> 2.1", optional: true},
-      {:postgrex, ">= 0.0.0", only: [:test]},
+      {:ecto, "~> 2.2 or ~> 3.0", optional: true},
```

Comment:
Do we need to keep this, or should we only be using `{:ecto_sql, "~> 3.0"}`? I think it was needed when working with release candidates, but now it should be fine to remove (as far as I understand). 

I'm getting that idea from reading [this blog post](http://blog.plataformatec.com.br/2018/10/a-sneak-peek-at-ecto-3-0-breaking-changes/), 

> If you are using Ecto with a SQL database, migrating to Ecto 3.0 will be very straight-forward. Instead of:
> ` {:ecto, "~> 2.2"}`
> You should list:
> `{:ecto_sql, "~> 3.0"}`

Reply:
Actually, I was not quite sure about which library we need to use in ExMachina. I just did a quick eyeballing of the codebase, I didn't find any code related to ecto_sql, so I chose ecto here.

I'll update the deps list if we do need `ecto_sql`.

Reply:
Will there be any chance that ExMachina is used in a project without a SQL database, with the same reason of the splition of Ecto.SQL?

Reply:
I would keep `ecto` as `{:ecto, "~> 3.0"}` and `ecto_sql` as `{:ecto_sql, "~> 3.0", optional: true}`, since you can technically use ex_machina to only build structs and not write to the DB. That said, I've had the same question in a few other dependencies where it's not as obvious.